mantle/cmd/kola/testiso.go

```diff
@@ -335,6 +335,23 @@ RequiredBy=coreos-installer.target
 # for target system
 RequiredBy=multi-user.target`, nmConnectionId, nmConnectionFile)
 
+// Unit to check that if there is any failed service
+var verifyNoFailedService = `[Unit]
+Description=TestISO Verify No Failed Service
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+After=rpm-ostree-fix-shadow-mode.service
```

Comment:
Ordering after this specific one feel too specific. If we add another unit in the future it won't be covered. 
Maybe we should add a systemd target, order after it and order all our unit before it. 
e.g. `coreos-sauce.target` :)